pkg/frontend/admin_openshiftcluster_kubernetesobjects_test.go

```diff
@@ -278,27 +282,27 @@ func TestValidateAdminKubernetesObjectsNonCustomer(t *testing.T) {
 			groupKind: "Valid-kind.openshift.io",
 			namespace: "openshift-/",
 			name:      "Valid-NAME-01",
-			wantErr:   "400: InvalidParameter: : The provided namespace 'openshift-/' is invalid.",
+			wantErr:   "403: Forbidden: : Access to the provided namespace 'openshift-/' is forbidden.",
```

Comment:
Why did it change from 400 to 403?

Reply:
Since we hard-coded the namespaces without any invalid name matching the regex in the `validate.go`, so the function `validateAdminKubernetesObjects()` will never be called and return  `400: InvalidParameter: : The provided namespace 'openshift-/' is invalid.` but only return `403: Forbidden: : Access to the provided namespace 'openshift-/' is forbidden.` from the function `validateAdminKubernetesObjectsNonCustomer()`